src/lib/exec-fetcher.test.js

```diff
@@ -61,5 +68,7 @@ packages.forEach(({ packageName, execName, execArgs, pattern, extension }) => {
 })
 
 test.afterEach(async t => {
-  await fs.remove(t.context.binPath)
+  await fs.rmdir(t.context.binPath, {
```

Comment:
⚠ the `recursive` flag for `rmdir` is experimental and was added only with Node.js v12, so probably out of scope for us, should we use `rimraf` here?

Reply:
Yes (or [`del`](https://github.com/sindresorhus/del) which is based on `rimraf`). The CLI minimum Node.js version is [`8.17.0`](https://github.com/netlify/cli/blob/d949407ef05060aaf2aae8bd227892e4646520ef/package.json#L14).

Reply:
`del` seems like a better alternative.